test/sql.test

```diff
@@ -41,17 +41,27 @@ namespace eval ::qcode::test {
     test sql_sort-1.0 {sql_sort} -setup {
     } -body {
         sql_sort name email user_code 
-    } -cleanup {} -result {name,email,user_code}
+    } -cleanup {} -result {"name","email","user_code"}
```

Comment:
Guess this doesn't apply cause you would just use the `-nulls` option for this.
Can you just quickly check MLA to make sure though?